packages/kiota_abstractions/lib/src/serialization/untyped_node.dart

```diff
@@ -0,0 +1,29 @@
+part of '../../kiota_abstractions.dart';
+
+/// Base class for untyped node.
+class UntypedNode implements Parsable {
+    UntypedNode();
+
+    /// Creates a new [UntypedNode] instance of the appropriate class
+    /// based on discriminator value.
+    factory UntypedNode.createFromDiscriminatorValue(ParseNode parseNode) {
+      return UntypedNode();
+    }
```

Comment:
What is this supposed to do?

Reply:
It creates an UntypedNode which is actually used when running json_parse_node_test.
The parameter is not used in this case.
It has the same implementation in dot-net and Java